mobile/examples/object_detections/iOS/ORTObjectDetection/ModelHandler.swift

```diff
@@ -0,0 +1,463 @@
+        let resultArray = formatResults(detectionBoxes: detectionBoxes, detectionClasses: detectionClasses, detectionScores: detectionScores, numDetections: numDetections, width: CGFloat(imageWidth), height: CGFloat(imageHeight))
+        
+        // Return ORT SessionRun result
+        return Result(processTimeMs: interval, inferences: resultArray)
+    }
+    
+    // MARK: - Helper Methods
+
+    // This method postprocesses the results including processing bounding boxes, sort detected scores, etc. and returns a result array
+    func formatResults(detectionBoxes: [Float32], detectionClasses: [Float32], detectionScores: [Float32], numDetections: Int, width: CGFloat, height: CGFloat) -> [Inference] {
+        
+        var resultsArray: [Inference] = []
+        
+        if (numDetections == 0) {
+            return resultsArray
+        }
+        
+        for i in 0...numDetections - 1 {
+            
+            let score = detectionScores[i]
+            
+            // Filter results with score < threshold.
+            guard score >= threshold else {
+                continue
+            }
+            
+            let detectionClassIndex = Int(detectionClasses[i])
+            let detectionClass = labels[detectionClassIndex + 1]
+            
+            var rect: CGRect = CGRect.zero
+            
+            // Translate the detected bounding box to CGRect.
+            rect.origin.y = CGFloat(detectionBoxes[4*i])
+            rect.origin.x = CGFloat(detectionBoxes[4*i+1])
+            rect.size.height = CGFloat(detectionBoxes[4*i+2]) - rect.origin.y
+            rect.size.width = CGFloat(detectionBoxes[4*i+3]) - rect.origin.x
+            
+            let newRect = rect.applying(CGAffineTransform(scaleX: width, y: height))
+            
+            let colorToAssign = colorForClass(withIndex: detectionClassIndex + 1)
+            let inference = Inference(score: score,
+                                      className: detectionClass,
+                                      rect: newRect,
+                                      displayColor: colorToAssign)
+            resultsArray.append(inference)
+        }
+        
+        // Sort results in descending order of confidence.
+        resultsArray.sort { (first, second) -> Bool in
+            return first.score  > second.score
+        }
+        
+        return resultsArray
+    }
+    
+    // This method preprocess the image by cropping pixel buffer to biggest square and scaling the cropped image to model dimensions.
+    private func preprocess(
+        ofSize size: CGSize,
+        _ buffer: CVPixelBuffer
+    ) -> CVPixelBuffer? {
+        
+        let imageWidth = CVPixelBufferGetWidth(buffer)
+        let imageHeight = CVPixelBufferGetHeight(buffer)
+        let pixelBufferType = CVPixelBufferGetPixelFormatType(buffer)
+        
+        assert(pixelBufferType == kCVPixelFormatType_32BGRA ||
+                pixelBufferType == kCVPixelFormatType_32ARGB)
```

Comment:
nit: formatting alignment
I tried swift-format and it seems to work pretty well
https://github.com/apple/swift-format